app/graphql/mutations/create_route_location.rb

```diff
@@ -0,0 +1,25 @@
+module Mutations
+  class CreateRouteLocation < BaseMutation
+
+    field :route_location, Types::RouteLocationType, null: false
+    field :errors, [String], null: false
+
+    argument :location_id, Types::LocationType, Integer, required: true
+    argument :route_id, Types::RouteType, Integer, required: true
+
+    def resolve(route_id:, location_id:, route_location:)
+      routeLocation = RouteLocation.save(route_id: route_id, location_id: location_id)
```

Comment:
You've been JS land too long ;) `Ruby` convention calls for `route_location` for var names.